pom.xml

```diff
@@ -328,31 +328,31 @@
                 <executions>
                     <execution>
                         <id>prepare-agent</id>
-			 
                       <goals><goal>prepare-agent</goal></goals>
-			  
                     </execution>
                     <execution>
                         <id>report</id>
                         <phase>test</phase>
-			 
                         <goals><goal>report</goal></goals>
-			  
                    </execution>
                </executions>
                <configuration>
 	               <rules>
 					     <rule>
-					     <element>BUNDLE</element>
-					     <limits>
-					     <limit>
-					     <counter>COMPLEXITY</counter>
-					     <value>COVEREDRATIO</value>
-					     <minimum>0.20</minimum>
-					     </limit>
+					       <element>BUNDLE</element>
+					       <limits>
+					           <limit>
+					               <counter>COMPLEXITY</counter>
+					               <value>COVEREDRATIO</value>
+					               <minimum>0.20</minimum>
+					       </limit>
 					     </limits>
 					     </rule>
 				     </rules>
+				     <excludes>
+                        <exclude>**/*KafkaStatusCheck.*</exclude>
```

Comment:
this one is actually not config, BUT KafkaAdminConfig is and KafkaProducerConfig is, which should make your numbers even out